hepdata/modules/email/api.py

```diff
@@ -310,6 +311,43 @@ def send_cookie_email(submission_participant,
                            message_body,
                            reply_to_address=coordinator.email)
 
+def send_reminder_email(submission_participant, record_information, version, show_detail=False):
+
+    site_url = current_app.config.get('SITE_URL', 'https://www.hepdata.net')
+    tables = []
+    hepsubmission = get_latest_hepsubmission(
+        publication_recid=record_information['recid']
+    )
+    coordinator = User.query.get(hepsubmission.coordinator)
+
+    reviewers_notified = submission_participant.user_account is not None
```

Comment:
Not sure about this.  I would just set `reviewers_notified = True` to get the correct text in `submission_status.html`.